CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v6.0.0](https://github.com/voxpupuli/puppet-icinga2/tree/v6.0.0) (2024-08-03)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-icinga2/compare/v5.0.1...v6.0.0)
+
+**Breaking changes:**
+
+- Remove EL7 support [\#774](https://github.com/voxpupuli/puppet-icinga2/pull/774) ([lbetz](https://github.com/lbetz))
+- Remove Debian Buster support [\#773](https://github.com/voxpupuli/puppet-icinga2/pull/773) ([lbetz](https://github.com/lbetz))
+
+**Implemented enhancements:**
+
+- Add Ubuntu noble \(24.04\) support [\#781](https://github.com/voxpupuli/puppet-icinga2/pull/781) ([lbetz](https://github.com/lbetz))
+- Add support for Fedora 40 [\#775](https://github.com/voxpupuli/puppet-icinga2/pull/775) ([lbetz](https://github.com/lbetz))
+- Fix SELinux and add feature::api::bind\_port support [\#772](https://github.com/voxpupuli/puppet-icinga2/pull/772) ([lbetz](https://github.com/lbetz))
```

Comment:
is that an enhancement or a bugfix?

Reply:
Both. Fixes missing sel_type of file resources. To change the listening port, it's a feature.